frontend/src/i18n/locales/de.yaml

```diff
@@ -437,7 +437,7 @@ manage:
         failed: Löschung fehlgeschlagen.
         warning: >
           Wenn Sie diese Seite löschen, werden automatisch alle direkten und
-          indirekten Unterseiten sowie alle Seiteninhalte all dieser Seiten
+          indirekten Unterseiten sowie alle Seiteninhalte dieser Seiten
```

Comment:
Maybe even 
```suggestion
          indirekten Unterseiten sowie alle Inhalte dieser Seiten
```

Reply:
Ye I suppose "all" is really not necessary. And +1 for "Inhalte".